Cargo.toml

```diff
@@ -23,6 +23,7 @@ members = [
 	"bee-ternary",
 	"bee-test",
 ]
+resolver = "2"
```

Comment:
Is this necessary ? It's the default since edition 2021

Reply:
Unfortunately yes, it fails to build with the `checksums` crate without it.